src/test/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/CreditorsWithinOneYearTransformerImplTests.java

```diff
@@ -130,6 +130,26 @@ void currentPeriodValueAddedToCreditorsWithinOneYearApiModel() {
 
         CreditorsWithinOneYear creditorsWithinOneYear = new CreditorsWithinOneYear();
 
+        AccrualsAndDeferredIncome accrualsAndDeferredIncome = new AccrualsAndDeferredIncome();
+        accrualsAndDeferredIncome.setCurrentAccrualsAndDeferredIncome(ACCRUALS_CURRENT);
+        creditorsWithinOneYear.setAccrualsAndDeferredIncome(accrualsAndDeferredIncome);
```

Comment:
We could the reduce code duplication by moving the instantiations to different method: 

`creditorsWithinOneYear.setAccrualsAndDeferredIncome(createAccrualsAndDeferredIncome());`
`creditorsWithinOneYear.setBankLoansAndOverdrafts(createBankLoansAndOverdrafts());`
`creditorsWithinOneYear.setFinanceLeasesAndHirePurchaseContracts(createFinanceLeasesAndHirePurchaseContracts());`
..etc

Reply:
Happy to do that...though we would save two lines for each field, but each method would add 4 lines, so it doesn't really help  (or am i misunderstanding):
```
private AccrualsAndDeferredIncome createCurrentAccrualsAndDeferredIncome() {
  AccrualsAndDeferredIncome accrualsAndDeferredIncome = new AccrualsAndDeferredIncome();
  accrualsAndDeferredIncome.setCurrentAccrualsAndDeferredIncome(ACCRUALS_CURRENT);
  return accrualsAndDeferredIncome;
}
```

Reply:
yes, you are right. I thought you had two tests setting the same information so that you could call it from each method; but I've just noticed they are different one for `current` and anbother for `previous`

```
void currentPeriodValueAddedToCreditorsWithinOneYearApiModel() {
...
    AccrualsAndDeferredIncome accrualsAndDeferredIncome = new AccrualsAndDeferredIncome();
    accrualsAndDeferredIncome.setCurrentAccrualsAndDeferredIncome(ACCRUALS_CURRENT);
    creditorsWithinOneYear.setAccrualsAndDeferredIncome(accrualsAndDeferredIncome);
...
}
```

```
void previousPeriodValueAddedToCreditorsWithinOneYearApiModel() {
 ...
    AccrualsAndDeferredIncome accrualsAndDeferredIncome = new AccrualsAndDeferredIncome();
    accrualsAndDeferredIncome.setPreviousAccrualsAndDeferredIncome(ACCRUALS_PREVIOUS);
    creditorsWithinOneYear.setAccrualsAndDeferredIncome(accrualsAndDeferredIncome);
...
}    
```